### PR TITLE
Add the IbmCloudGlobalTagging gem

### PR DESCRIPTION
**Pull request by agrare**
This is required for https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/48

### CODE REVIEW

File: gems/ibm_cloud_global_tagging/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Questions
+If you are having problems using the APIs or have a question about IBM Cloud services,
+please ask a question at
+[Stack Overflow](http://stackoverflow.com/questions/ask?tags=ibm-cloud).
+
+# Issues
+If you encounter an issue with the project, you are welcome to submit a
+[bug report](https://github.com/IBM-Cloud/global_tagging-ruby-sdk/issues).
```
Comment (jaywcarman) [inline gems/ibm_cloud_global_tagging/CONTRIBUTING.md:8]: I'm guessing 'IBM-Cloud/global_tagging-ruby-sdk' will be the future home?

**Review by jaywcarman — COMMENTED** [review]
Are we planning to move this gem into https://github.com/IBM-Cloud/global_tagging-ruby-sdk ? That is the repo URL used in the docs.

Other than that looks great.

File: gems/ibm_cloud_global_tagging/CONTRIBUTING.md (hunk 1 above)
Comment (agrare) [reply gems/ibm_cloud_global_tagging/CONTRIBUTING.md]: :+1: good catch, fixed

**Review by jaywcarman — APPROVED** [review]
LGTM

**Pull request merged**